<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EcranPrincipal</class>
 <widget class="QWidget" name="EcranPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelNom">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Bienvenue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>EasyTicket</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCreationTicket">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Créer un ticket</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listeTickets">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>611</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDeco">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Déconnexion</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
